tensorflow/python/keras/callbacks_test.py

```diff
@@ -1283,6 +1284,27 @@ def target():
       t.join()
       assert not t.is_alive()
 
+  @test.mock.patch.object(requests, 'post')
```

Comment:
If possible, make this test pass unconditially if `requests` is `None`.

Otherwise, we might need to drop the tests or rewrite them to not use `requests`.

Reply:
Added a guard for this.